packages/turf-points-within-polygon/index.js

```diff
@@ -41,11 +41,28 @@ function pointsWithinPolygon(points, polygons) {
   var results = [];
   featureEach(points, function (point) {
     var contained = false;
-    geomEach(polygons, function (polygon) {
-      if (pointInPolygon(point, polygon)) contained = true;
-    });
-    if (contained) {
-      results.push(point);
+    if (point.geometry.type === "Point") {
+      geomEach(polygons, function (polygon) {
+        if (pointInPolygon(point, polygon)) contained = true;
+      });
+      if (contained) {
+        results.push(point);
+      }
+    } else if (point.geometry.type === "MultiPoint") {
+      var pointsWithin = [];
```

Comment:
Just an fyi, I tried to use `let` over `var` here out of habit to narrow the scope to the code block, but the CI linter complained that `let` is reserved.   I was surprised VSCode didn't warn me, I have the eslint module turned on.  @mfedderly I thought I saw you were taking suggestions for relaxing linter rules, perhaps this is one.

https://github.com/Turfjs/turf/runs/2938648986?check_suite_focus=true
```
          ES-Check Error:
          ----
          · erroring file: packages/turf-points-within-polygon/dist/js/index.js
          · error: SyntaxError: The keyword 'let' is reserved (58:6)
```

Reply:
In this case you aren't hitting eslint rules, but rather the es-check rule. My understanding is that you should not ship non-es5 code as part of your packages. Although the typescript packages in the repo allow us to use the newer es6 features, we do not currently have a transpilation pipeline set up for the javascript packages. Ideally this check is still left in to make sure none of our dependencies accidentally ship es6 as well.

If my knowledge is outdated (caniuse.com says almost 95% of all clients support the let keyword at this point), then we should figure out what we want our officially supported targets to be and go from there.

Reply:
Makes sense.  I guess the presence of `const` in the tests that made me think it was okay.  I can see es-check is only run on the dist